hackathon/forms.py

```diff
@@ -74,6 +74,7 @@ class HackathonForm(forms.ModelForm):
     team_size = forms.IntegerField(
         label="Team Size",
         required=True,
+        widget=forms.TextInput(attrs={'min':1,'max': 6,'type': 'number'})
```

Comment:
Correct me if I'm wrong, but is this `team_size` can have a minimum of `1` person on a team? Shouldn't we put a min to 3 perhaps?

Reply:
Changed as suggested.